scripts/run.sh

```diff
@@ -190,7 +190,7 @@ fi
     --part.minimalKineticEnergy 1*TeV \
     --filter.tracker 'edep0' \
     --hepmc3.useHepMC3 ${USEHEPMC3:-true} \
-    --compactFile ${DETECTOR_PATH}/${DETECTOR_CONFIG}.xml \
+    --compactFile ${DETECTOR_PATH}/${DETECTOR_CONFIG}_${EBEAM}_${PBEAM}.xml \
```

Comment:
Can we stick to the following?
```suggestion
    --compactFile ${DETECTOR_PATH}/${DETECTOR_CONFIG}_${EBEAM}x${PBEAM}.xml \
```
and fall back to just `${DETECTOR_CONFIG}.xml` if no beam conditions defined (as in default load of epic setup.sh)?

Reply:
ermm then we need to change the epic xinja template